app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,52 @@
+class RestaurantsController < ApplicationController
+  before_action :set_restaurant, only: [:show, :update, :destroy]
+
+  # GET /restaurants
+  def index
+    @restaurants = Restaurant.all
+
+    render json: @restaurants
+  end
+
+  # GET /restaurants/1
+  def show
+    @restaurants = Restaurant.find(params[:id])
+    render json: @restaurant
+  end
+
+  # POST /restaurants
+  def create
+    @restaurant = Restaurant.new(restaurant_params)
+
+    if @restaurant.save
+      render json: @restaurant, status: :created, location: @restaurant, notice:'Restaurant was successfully created.'
+    else
+      render json: @restaurant.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /restaurants/1
+  def update
+    if @restaurant.update(restaurant_params)
+      render json: @restaurant, notice: 'Restaurant was successfully updated.'
+    else
+      render json: @restaurant.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /restaurants/1
+  def destroy
+    @restaurant.destroy
```

Comment:
shouldn't it also return some sort of status?

Reply:
no?